platform/mellanox/docker-syncd-mlnx.mk

```diff
@@ -17,7 +17,39 @@
 # docker image for mlnx syncd
 
 DOCKER_SYNCD_PLATFORM_CODE = mlnx
-include $(PLATFORM_PATH)/../template/docker-syncd-base.mk
+
```

Comment:
Please move this section to the template and have if condition for bullseye there.

Reply:
I am not able to do that I don't think. 

This logic depends on whether a given platform's syncd supports bullseye, not if the current build environment is bullseye (I'm not sure that we have that variable at this point in the build process) 

This code itself is setting up that dependency here. We need to run the code that depends syncd on bullseye for our code and only our code.

The appropriate thing to do that would most closely mirror your suggestion is to add an if statement checking what platform we are building for but I figured we did not want such logic in the template.

Reply:
Understood. Instead can we create docker-syncd-base-bullseye.mk and have this as common code there? Later when all vendors are migrated we can ask MSFT to move it docker-syncd-base.mk

Reply:
I see. I think this will work fine.